test/com/google/enterprise/adaptor/filenet/DocumentTraverserTest.java

```diff
@@ -408,6 +433,156 @@ public void testGetDocContent() throws Exception {
   }
 
   @Test
+  public void testGetDocContent_includedMetadata() throws Exception {
+    options = TestObjectFactory.newConfigOptions(
+        ImmutableMap.of("filenet.includedMetadata", "foo, bar"));
+
+    String id = "{AAAAAAAA-0000-0000-0000-000000000000}";
+    DocId docId = newDocId(new Id(id));
+    MockObjectStore os = getObjectStore();
+    mockDocument(os, id, DOCUMENT_TIMESTAMP, RELEASED, 42d, "text/plain",
+        ImmutableList.<Property>of(
+            mockProperty("foo", "fooValue"),
+            mockProperty("bar", "barValue")),
+        new AccessPermissionListMock(),
+        new ActiveMarkingListMock());
+
+    DocumentTraverser traverser = new DocumentTraverser(options);
+    Request request = new MockRequest(docId);
+    RecordingResponse response = new RecordingResponse();
+    traverser.getDocContent(new Id(id), request, response);
+
+    assertEquals(
+        new Metadata(
+            ImmutableMap.of("foo", "fooValue", "bar", "barValue")
+            .entrySet()),
```

Comment:
It was mostly copy/paste consistency, but I already broke that consistency in another test.